continuous_integration/recipe/meta.yaml

```diff
@@ -15,12 +15,13 @@ build:
   noarch: python
   string: py_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
   script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - dask = dask.__main__:main
 
 requirements:
   host:
     - python >=3.8
     - pip
-    - setuptools
```

Comment:
Unrelated to the CLI changes, but noticed that `setuptools` got dropped as a host dependency from the conda-forge recipe

Reply:
Yeah it is a dependency of `pip`. So always gets pulled in